sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1141,11 +1141,11 @@ class Lasso(ElasticNet):
     Parameters
     ----------
     alpha : float, default=1.0
-        Constant that multiplies the L1 term. Defaults to 1.0.
-        ``alpha = 0`` is equivalent to an ordinary least square, solved
-        by the :class:`LinearRegression` object. For numerical
-        reasons, using ``alpha = 0`` with the ``Lasso`` object is not advised.
-        Given this, you should use the :class:`LinearRegression` object.
+        Constant that multiplies the L2 term. Defaults to 1.0.
```

Comment:
The current formulation is correct. Lasso is L1 prenalized
```suggestion
        Constant that multiplies the L1 term. Defaults to 1.0.
```

Reply:
Good catch!